crates/bevy_pbr/src/wireframe.rs

```diff
@@ -1,30 +1,28 @@
-use crate::{
-    DrawMesh, MeshPipeline, MeshPipelineKey, RenderMeshInstance, RenderMeshInstances,
-    SetMeshBindGroup, SetMeshViewBindGroup,
-};
-use bevy_app::Plugin;
-use bevy_asset::{load_internal_asset, Handle};
-use bevy_core_pipeline::core_3d::Opaque3d;
-use bevy_derive::{Deref, DerefMut};
-use bevy_ecs::{prelude::*, reflect::ReflectComponent};
-use bevy_reflect::{std_traits::ReflectDefault, Reflect};
+use crate::{Material, MaterialPipeline, MaterialPipelineKey, MaterialPlugin};
+use bevy_app::{Plugin, Startup, Update};
+use bevy_asset::{load_internal_asset, Asset, Assets, Handle};
+use bevy_ecs::prelude::*;
+use bevy_reflect::{std_traits::ReflectDefault, Reflect, TypePath, TypeUuid};
 use bevy_render::{
-    extract_resource::{ExtractResource, ExtractResourcePlugin},
+    extract_resource::ExtractResource,
     mesh::{Mesh, MeshVertexBufferLayout},
-    render_asset::RenderAssets,
-    render_phase::{AddRenderCommand, DrawFunctions, RenderPhase, SetItemPipeline},
+    prelude::Shader,
     render_resource::{
-        PipelineCache, PolygonMode, RenderPipelineDescriptor, Shader, SpecializedMeshPipeline,
-        SpecializedMeshPipelineError, SpecializedMeshPipelines,
+        AsBindGroup, PolygonMode, RenderPipelineDescriptor, ShaderRef, SpecializedMeshPipelineError,
     },
-    view::{ExtractedView, Msaa, VisibleEntities},
-    RenderApp, RenderSet,
 };
-use bevy_render::{Extract, ExtractSchedule, Render};
-use bevy_utils::{tracing::error, EntityHashSet};
 
 pub const WIREFRAME_SHADER_HANDLE: Handle<Shader> = Handle::weak_from_u128(192598014480025766);
 
+/// A [`Plugin`] that draws wireframes.
+///
+/// Wireframes currently do not work when using webgl or webgpu.
+/// Supported platforms:
+/// - DX12
+/// - Vulkan
+/// - Metal
```

Comment:
most other places in our docs, "platforms" means linux, windows, macos, wasm

Reply:
```suggestion
/// Wireframes currently do not work when using webgl or webgpu.
/// Supported rendering backends:
/// - DX12
/// - Vulkan
/// - Metal
```